hie.yaml

```diff
@@ -0,0 +1,225 @@
+cradle:
+  multi:
+  - path: ./language-plutus-core/generators
+    config:
+      cradle:
+        cabal:
+          component: "lib:language-plutus-core"
+  - path: ./language-plutus-core/prelude
+    config:
+      cradle:
+        cabal:
+          component: "lib:language-plutus-core"
+  - path: ./language-plutus-core/src
+    config:
+      cradle:
+        cabal:
+          component: "lib:language-plutus-core"
+  - path: ./language-plutus-core/test
+    config:
+      cradle:
+        cabal:
+          component: "test:language-plutus-core"
+  - path: ./plutus-core-interpreter/src
+    config:
+      cradle:
+        cabal:
+          component: "lib:plutus-core-interpreter"
+  - path: ./plutus-core-interpreter/test
+    config:
+      cradle:
+        cabal:
+          component: "test:plutus-core-interpreter"
```

Comment:
Is this valid component syntax? I can't `cabal v2-build test:plutus-core-interpreter`. I can do `cabal v2-build plutus-core-interpreter:plutus-core-interpreter-test` - I think you need the name of the actual test component.

Reply:
It does seem to work for the `lib` case, though :thinking:

Reply:
Yup, you're right.

```
Failed to parse result of calling cabal
cabal: Internal error in target matching. It should always be possible to find
a syntax that's sufficiently qualified to give an unambiguous match. However
when matching 'test:language-plutus-core' we found test:language-plutus-core
(unknown-component) which does not have an unambiguous syntax. The possible
syntax and the targets they match are as follows:
'test:language-plutus-core' which matches test:language-plutus-core
(unknown-component), :pkg:test:lib:test:file:language-plutus-core
(unknown-file)
```
Will fix.

Reply:
I fixed it, but it doesn't seem to get further than "Initializing Cradle..." :-(

Reply:
Hm, weird. I'd be happy with either just getting rid of the cradles for test components if they don't work, or leaving them in if we think the definition is correct and it'll get fixed by later HIE work.

Reply:
Fixed it.